tests/test_arbitrary_scripts.py

```diff
@@ -83,6 +82,12 @@ def test_arbitrary_scripts_allowed(backend, firmware, navigator, test_name):
                                                       "Hold to sign",
                                                       ROOT_SCREENSHOT_PATH,
                                                       test_name + "_1")
+        elif backend.firmware.device == "flex":
+            navigator.navigate_until_text_and_compare(NavInsID.SWIPE_CENTER_TO_LEFT,
+                                                      [NavInsID.USE_CASE_REVIEW_CONFIRM, NavInsID.USE_CASE_STATUS_DISMISS],
+                                                      "Hold to sign",
+                                                      ROOT_SCREENSHOT_PATH,
+                                                      test_name + "_1")
```

Comment:
could be probably merged in one if 

```
if backend.firmware.is_nano:  (instead of startwith....)
...
elif:
same code for Stax as for Flex (swipe is also managed in ragger for stax)
```